orchestra/assignment_policies/simple.py

```diff
@@ -0,0 +1,61 @@
+from orchestra.core.errors import AssignmentPolicyError
+from orchestra.models import Task
+from orchestra.utils.task_lifecycle import assign_task
+from orchestra.utils.task_properties import assignment_history
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def anyone_certified(task, related_steps):
```

Comment:
ah i see i thought `kwargs` was for more specific params but `related_steps` was required by all :)